site/public/js/server.js

```diff
@@ -474,6 +475,21 @@ function editFilePathRecommendations() {
     registerSelect2Widget('new-file-name', 'material-edit-form');
 }
 
+function showPopup(selector) {
+    $(selector).css('display', 'block');
+    document.body.classList.add('no-scroll');
+}
+
+/* eslint-disable-next-line no-unused-vars */
+function closePopup(id) {
```

Comment:
Instead of ignoring the error inline like this, the name of the function should be added to the list at the top of the file.